tests/snapshots/html_test__html_doc_files_rewrite-16.snap

```diff
@@ -231,7 +231,7 @@ Type Aliases</h2></div><div class="namespaceSection"><div class="namespaceItem"
 <span class="hover:bg-Interface/15">Hello</span>
               </a>
             </li></ul><a class="flex items-center gap-0.5" href="..&#x2F;.&#x2F;all_symbols.html">
-            <span class="leading-none">view all 6 symbols</span><svg
+            <span class="leading-none">view all 8 symbols</span><svg
```

Comment:
All symbols page only shows 6 symbols (because we intentionally don't list `A` and `B`). Should this number exclude private and internal symbols?

Reply:
hm, you are right, will update that